docs/cell_census_schema_0.0.1.md

```diff
@@ -0,0 +1,853 @@
+    <td>x</td>
+    <td>x</td>
+  </tr>
+  <tr>
+    <td>[Homo sapiens|Mus musculus]</td>
+    <td>sex</td>
+    <td>…</td>
+    <td>…</td>
+    <td>x</td>
+    <td>x</td>
+  </tr>
+  <tr>
+    <td>[Homo sapiens|Mus musculus]</td>
+    <td>sex</td>
+    <td>sex_N</td>
+    <td>PATO:XXXXX</td>
+    <td>x</td>
+    <td>x</td>
+  </tr>
+  <tr>
+    <td>[Homo sapiens|Mus musculus]</td>
+    <td>suspension_type</td>
+    <td>suspension_type_a</td>
+    <td>na</td>
+    <td>x</td>
+    <td>x</td>
+  </tr>
+  <tr>
+    <td>[Homo sapiens|Mus musculus]</td>
+    <td>suspension_type</td>
+    <td>…</td>
+    <td>…</td>
+    <td>x</td>
+    <td>x</td>
+  </tr>
+  <tr>
+    <td>[Homo sapiens|Mus musculus]</td>
+    <td>suspension_type</td>
+    <td>suspension_type_N</td>
+    <td>na</td>
+    <td>x</td>
+    <td>x</td>
+  </tr>
+</tbody>
+</table>
+
+### Cell Census Data – `census_obj[“census_data”][organism]` – `SOMAExperiment`
+
+Data for *Homo sapiens* MUST be stored as a `SOMAExperiment` in `census_obj[“homo_sapiens”]`.
+
+Data for *Mus musculus* MUST be stored as a `SOMAExperiment` in `census_obj[“mus_musculus”]`.
+
+For each organism the `SOMAExperiment` MUST contain the following:
+
+* Cell metadata – `census_obj[“census_data”][organism].obs` – `SOMADataFrame`
+* Data  –  `census_obj[“census_data”][organism].ms` – `SOMACollection`. This `SOMACollection` MUST only contain one `SOMAMeasurement` in `census_obj[“census_data”][organism].ms[“RNA”]` with the following:
+	* Matrix  data –  `census_obj[“census_data”][organism].ms[“RNA”].X` – `SOMACollection`. It MUST contain exactly one layer: 
+		* Count matrix – `census_obj[“census_data”][organism].ms[“RNA”].X[“raw”]` – `SOMASparseNdArray`
+	* Feature metadata – `census_obj[“census_data”][organism].ms[“RNA”].var` – `SOMAIndexedDataFrame`
+	* Feature dataset presence matrix – `census_obj[“census_data”][organism].ms[“RNA”].varp[“dataset_presence_matrix”]` – `SOMASparseNdArray`
+
+
+For each organism the `SOMAExperiment` SHOULD NOT contain the following:
+
+* Elements of `census_obj[“census_data”][organism].ms[“RNA”]`: 
+	* `var_ms`
+	* `obsm` 
+	* `obsp`
+	* `obs_ms`
+
+#### Matrix Data, count (raw) matrix – `census_obj[“census_data”][organism].ms[“RNA”].X[“raw”]` – `SOMASparseNdArray`
+
+Per the CELLxGENE dataset schema, [all RNA assays MUST include UMI or read counts](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/3.0.0/schema.md#x-matrix-layers). These counts MUST be encoded as `float32` in this `SOMASparseNdArray`. `0` values in datasets with sparse matrices MUST NOT be explicitly encoded as `0`, they MUST be properly stored in sparse format. 
+
+
+#### Feature metadata – `census_obj[“census_data”][organism].ms[“RNA”].var` – `SOMADataFrame`
+
+Only features with [`adata.var.feature_biotype == “gene”`](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/3.0.0/schema.md#feature_biotype) (as defined the CELLxGENE dataset schema) MUST be included.
```

Comment:
editorial: same as previous, this appears to allow other values.  For clarity:   "'gene' MUST be included, other values MUST NOT be included"